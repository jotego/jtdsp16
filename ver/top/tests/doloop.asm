y=1
do 127 {
    a1=a1+y
    a0=a0+y
}

end:
goto end
