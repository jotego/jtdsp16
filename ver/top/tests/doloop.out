r0=0x0000
r1=0x0000
r2=0x0000
r3=0x0000
rb=0x0000
re=0x0000
j =0x0000
k =0x0000
pr=0x0000
pt=0x0000
i =0x0000
a0=0x0007f0000
a1=0x0007f0000
x=0x0000
y=0x00010000
p=0x00000000
c0=0x0000
c1=0x0000
c2=0x0000
auc=0x0000
psw=0x0000
